src/player.cpp

```diff
@@ -1031,7 +1031,7 @@ void Player::sendRemoveContainerItem(const Container* container, uint16_t slot)
 		uint16_t& firstIndex = openContainer.index;
 		if (firstIndex > 0 && firstIndex >= container->size() - 1) {
 			firstIndex -= container->capacity();
-			sendContainer(it.first, container, false, firstIndex);
+			sendContainer(it.first, container, firstIndex);
```

Comment:
Sending `false` here is incorrect; it should be validated first (`hasContainerParent`)! If I'm in a Inbox and remove the last item from a page, current Inbox contains the DepotLocker as the parent. Was it always sending `false` previously...